src/passwd.c

```diff
@@ -748,6 +750,13 @@ int main (int argc, char **argv)
 	process_root_flag ("-R", argc, argv);
 	prefix = process_prefix_flag ("-P", argc, argv);
 
+#ifdef USE_PAM
+	if (prefix[0]) {
+		use_pam = false;
+		do_update_age = true;
+	}
+#endif /* USE_PAM */
```

Comment:
Do you really need to build this conditionally?

Reply:
Fixed.